sql/sql_show.cc

```diff
@@ -10403,6 +10695,8 @@ ST_SCHEMA_TABLE schema_tables[]=
    fill_status, make_old_format, 0, 0, -1, 0, 0},
   {"SESSION_VARIABLES", Show::variables_fields_info, 0,
    fill_variables, make_old_format, 0, 0, -1, 0, 0},
+  {"SLAVE_STATUS", Show::slave_status_info, 0,
```

Comment:
Oh one more thing - Kristian mentioned we should have both `slave status` and `replica status`; where the later is an alias of the former. It looks like `replica` was scrapped altogether in favor of `slave`, but we need both.

Reply:
Will do and update MDEV.